### PR TITLE
cleanup utility 

### PR DESCRIPTION
**Pull request by sayakpaul**
We should clean up the `tmp` directory as it can cause our node to drain out. 

Check internal thread: https://huggingface.slack.com/archives/C043JTYE1MJ/p1726211997401189.

### CODE REVIEW

File: inference/utils.py [hunk 1]
```diff
@@ -42,3 +45,36 @@ def format_value(value):
 
     filtered_table = {k: format_value(v) for k, v in results.items()}
     print(tabulate([filtered_table], headers="keys", tablefmt="pipe", stralign="center"))
+
+
+def get_current_user():
```
Comment (sayakpaul) [inline inference/utils.py:50]: Courtesy of Hugo.

**Review by a-r-r-o-w — APPROVED** [review]
Ah interesting, thanks!

**Comment by a-r-r-o-w** [comment]
FYI: My latest run deleted everything from `/tmp/aryan` including compiled output code, previous triton files, etc. and killed my tmux sessions because tmux stores its session info in a random file like: `/tmp/tmpj4txyw6vwandb`.

Would not recommend this if you use /tmp for storing stuff like installers, etc. (my pyenv installer was in /tmp/aryan too which got nuked as well)

**Comment by sayakpaul** [comment]
tmux sessions are fine. Installers should never go to `/tmp`. It should be under the `/fsx` volume. 

For the compiled code output, we can always set `TORCH_LOGS_OUT`.

**Pull request merged**